helpers/error_handler/error_message.go

```diff
@@ -11,13 +11,15 @@ const (
 	commonFieldError        = "%s is not valid"
 	emailFormatError        = "invalid email format"
 	emailDuplicateError     = "email already exists"
+	invalidInputError       = "invalid input"
```

Comment:
This message is still too generic and not very helpful 😵 It can work for now but I am still trying to find better ways to handle errors with form. See my comment below.

Reply:
Replied [here](https://github.com/gutakk/go-google-scraper/pull/32#discussion_r548437819)